src/Education/XamlGenerator/RuleHelpXamlTranslator.cs

```diff
@@ -23,62 +23,60 @@
 using System.Diagnostics;
 using System.IO;
 using System.Text;
-using System.Windows.Documents;
-using System.Windows.Markup;
 using System.Xml;
-using SonarLint.VisualStudio.Rules;
 
 namespace SonarLint.VisualStudio.Education.XamlGenerator
```

Comment:
Because I have changed the filename of RuleHelpXamlBuilder, git decided to match RuleHelpXamlTranslator.cs to the Old file.